### ISSUE TITLE
Feature Request: Pronunciation of words

### ISSUE THREAD

**Issue by gpauvert** [tragram/finnish-inflexion-drill#1]
First of all I would like to say what a great resource you have made here! It is clean and simple and quite intuitive. A potential feature could be the pronunciation of the words.

**Comment by tragram** [comment]
Hey!
Thanks for the feedback. By pronunciation you mean audio?

**Comment by gpauvert** [comment]
yeah perhaps using the the [gTTS package](https://pypi.org/project/gTTS/) seeing as you are already using python

**Comment by tragram** [comment]
Well, my view on that is that Finnish has, unlike English, pretty simple pronunciation rules, so the added value of this is not too great.
In addition to that, generating all the sounds in Python would increase the size of the website significantly (it is static, so it is not possible to only send the audio when requested).
